capstone-app/src/app/app.component.html

```diff
@@ -1,4 +1,8 @@
 <h1>{{title}}</h1>
-<app-map></app-map>
+<app-map
+  (activeMarkerOutput)="updateActiveMarker($event)"
+  (markerDataOutput)="updateMarkerData($event)"
+  (cityOutput)="cityUpdate($event)">
+</app-map>
 <app-photos-section [city]=(city)></app-photos-section>
```

Comment:
I've never seen this parentheses notation in place of the quotes, interesting. Is there a reason for this?

Reply:
Personally I like it because it mirrors syntax for event listeners such as click

Reply:
Nope, let me change that!

Reply:
Hey Nicholas what’s the alternative syntax? The angular website https://angular.io/api/core/EventEmitter uses ours

Reply:
So, I guess this is actually how we meant to do it.. the angular site uses this format.
@NicholasCobbFromGoogle

Reply:
I just would have expected it to be more like
<app-photos-section [city]="city">

Reply:
Oh, should I change it? @NicholasCobbFromGoogle